src/components/Comments/ReplyCommentItem.js

```diff
@@ -316,11 +379,18 @@ const ReplyCommentItem = ({
             )}
           </ButtonHightlight>
         </TouchableOpacity>
-        {totalVote !== 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
+        {<Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for displaying the total vote count has been removed. If this is intentional and you always want to display the vote count even if it's zero, then this change is fine.



<!-- This is an auto-generated comment by OSS CodeRabbit -->